packages/sdk/src/embed/react.tsx

```diff
@@ -277,6 +276,55 @@ function useIframeDimensionsWatcher(iframeUri: string) {
 
   return {
     dimensions,
-    iframeRef,
   };
 }
+
+/**
+ * On mobile the uri scheme cannot be triggered directly within the iframe, so we had to intercept the message and open the link on the container level.
+ *
+ * p.s. we had to hack rainbowkit to make expose such window message event, check pnpm patch for details.
+ */
+function useWalletButtonInterceptor(embedUri: string) {
+  useEffect(() => {
+    if (typeof window === "undefined") {
+      return;
+    }
+
+    const origin = new URL(embedUri).origin;
+
+    const handleMessage = (event: MessageEvent) => {
+      if (
+        event.data.type !== "rainbowkit-wallet-button-mobile-clicked" ||
+        origin !== event.origin
+      ) {
+        return;
+      }
+
+      const mobileUri = event.data.uri;
+
+      if (
+        mobileUri.startsWith("javascript:") ||
+        mobileUri.startsWith("data:")
```

Comment:
I think this should be lower cased first and then checked using `startsWith`.

Reply:
fixed